notebooks/official/migration/sdk-hyperparameter-tuning.ipynb

```diff
@@ -543,7 +457,9 @@
         "\n",
         "The files `setup.cfg` and `setup.py` are the instructions for installing the package into the operating environment of the Docker image.\n",
         "\n",
-        "The file `trainer/task.py` is the Python script for executing the custom training job. *Note*, when we referred to it in the worker pool specification, we replace the directory slash with a dot (`trainer.task`) and dropped the file suffix (`.py`).\n",
+        "The file `trainer/task.py` is the Python script for executing the custom training job. \n",
+        "\n",
+        "*Note* : when referred to it in the worker pool specification, replaced the directory slash with a dot (`trainer.task`) and dropped the file suffix (`.py`).\n",
```

Comment:
Change Note:
- To: "Note*:  When `trainer/task.py` is referred to in the worker pool specification, the directory slash is replaced with a dot and the file suffix (`.py`) is dropped (`trainer.task`)."